### DISCUSSION TITLE
Use windows 11 logo on windows 10 taskbar

### DISCUSSION THREAD

**Discussion by DelvinNuryadi** [valinet/ExplorerPatcher#375] [Ideas]
i dont like windows 10 start icon, i suggest to add feature to change the start logo

**Comment by randomguyreviewing** [comment]
Same. I would love to see that update

**Comment by sisericm** [comment]
@DelvinNuryadi 
Check #142 & #339

**Comment by valinet** [comment]
Duplicate, and not an issue, but a feature request, so belongs to discussions.
